backend/routes_kit_transfers.py

```
@@ -65,22 +66,90 @@ def create_transfer():
+        # Update source location - remove item from source
+        if data['from_location_type'] == 'kit' and source_item:
+            source_item.quantity -= quantity
+            if source_item.quantity <= 0:
+                # Remove the item completely if quantity is 0
+                db.session.delete(source_item)
+
+        # Update destination location - add item to destination
+        if data['to_location_type'] == 'kit':
+            dest_kit = Kit.query.get(data['to_location_id'])
+            if not dest_kit:
+                raise ValidationError('Destination kit not found')
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)